e2e/features/time-extension.feature

```diff
@@ -41,5 +41,6 @@ Feature: Review time extension
 
     When I click the `Submit` button
     Then I should see the text `has been updated with event: Review time extension`
+    And I should see `31 Dec 2018` for the `By what date must they comply?` answer
```

Comment:
Just double checking if the time extension request was requested on the 2020-01-01 according to the step on:     `And the appellant submits a time extension`  doesn't this date feel a bit wrong? it would mean that the time extension was fired after the direction due date  (1 year after to be exact) Since we don't have checks or business logic yet to handle this case the tests are correct but might break in the future if we do add logic that prevents a user from submitting time extensions after due date has passed.
Perhaps choosing a date with 1-week offset might be better :D